tasks/main.yml

```diff
@@ -30,6 +30,7 @@
 
 - include: keepalived_selinux.yml
   when:
+    - keepalived_selinux_compile_rules
```

Comment:
that is always not falsy by default. Could you explain why do we need this? Maybe we could move this to ``| length > 0 `` for those making the list an empty list, if that's what you want to achieve? (I think it has a clearer meaning when reading the code)

Reply:
I know it is not empty by default. Nevertheless I do not want to have any additional SELinux packages (i.e. libselinux-devel) installed when this list is empty (or possibly False/None/...).

I am OK with any of:
- `keepalived_selinux_compile_rules`
- `not not keepalived_selinux_compile_rules`
- `keepalived_selinux_compile_rules | bool`
- `keepalived_selinux_compile_rules | length > 0`

Choose the one you prefer and I'll update this PR.

Reply:
``keepalived_selinux_compile_rules | length > 0`` would be my preference.

If the list is non-empty, it would be running, and you can still override with an empty list (with []) to skip in the when condition.